libs/ui-toolkit/form-text/README.md

```diff
@@ -0,0 +1,9 @@
+# ui-toolkit-form-text
```

Comment:
This file doesn't add much information and other libraries don't have it, so it could be removed